sources/packages/backend/libs/services/src/constants/disbursements.constants.ts

```diff
@@ -31,7 +31,9 @@ export const BC_FUNDING_TYPES = [
  * that is represented by this code.
  */
 export const BC_TOTAL_GRANT_AWARD_CODE = "BCSG";
+
```

Comment:
please remove the additional line

Reply:
When they were grouped it gave the impression that the previous comment was to the entire block of `constants` when it was not.
I can remove it if the Team understands that the way it was was clear.